diesel_infer_schema/src/data_structures.rs

```diff
@@ -27,6 +27,7 @@ pub struct ColumnType {
 pub struct ColumnDefinition {
     pub name: String,
     pub ty: ColumnType,
+    pub new_name: Option<String>,
```

Comment:
`new_name` feels weird to me. How about we call it `rust_name`, and rename the `name` field to `sql_name`?